packages/web/components/Landing/Build.tsx

```diff
@@ -54,16 +54,25 @@ export const Build: React.FC = () => {
         >
           <Text fontSize={{ base: 'xl', '2xl': '3xl' }}>
             People are waking up to the{' '}
-            <strong>world-shaping potential of Web3 technologies</strong>.
+            <Text as="strong" className="gradient-text">
+              world-shaping potential of Web3 technologies
+            </Text>
+            .
           </Text>
 
           <Text pt={{ base: 4, md: 8 }}>
             They are grabbing the opportunity to{' '}
-            <strong>build the future</strong> they want to live in.
+            <Text as="strong" className="gradient-text">
+              build the future
+            </Text>{' '}
```

Comment:
```suggestion
            They are grabbing the opportunity to
            <Text as="strong" mx={2} className="gradient-text">
              build the future
            </Text>
```

Reply:
I'd guessed that the {' '} is because trailing spaces are stripped and the words run together. I'll read up on that.

Reply:
Seems it is like that, ref.: https://stackoverflow.com/questions/38547558/why-do-people-put-in-their-react-jsx

> This is used to put an explicit space in a text block, since leading and trailing spaces are ignored at compile/transformation time when there is another tag.

IMO the {' '} has been used in the code to make it obvious that there is a space after the Text component. It looks like it could be left as a plain space and it would still work.

I think using a plain space or the {' '} is cleaner than adding margins.

Reply:
I did some testing on this.

ml={2} gives a space between words that is roughly 2x the regular space
ml="0.25em" gives a space between words that is roughly the same size as a regular space
(see the screenshot)

![image](https://user-images.githubusercontent.com/92962228/181165251-87b8a912-a844-4239-83fa-9ad356ac0f27.png)

Using ml doesn't seem to affect the reflow of text. The ml doesn't add the space added if the text has flowed down to the beginning of the next line e.g.

It displays as
```
hard to navigate
```
not
```
  hard to navigate
```

So using ml={0.25em} instead of the {' '} would work.

My preference is for the {' '} because it's clear what's happening.